string-manipulation-2/src/last-chars.js

```diff
@@ -0,0 +1,13 @@
+/* exported lastChars */
+function lastChars(length, string) {
+  let str = '';
+  for (let i = string.length - length; i <= string.length; i++) {
+    if (i < string.length) {
+      str += string[i];
+    } else if (length > string.length) {
+      return 'LearningFuze';
+    }
+  }
+  console.log(str);
```

Comment:
This could also be done with the `slice` method and calculating the index to start at by subtracting the string length by `length`